tcti/tcti_socket.map

```diff
@@ -2,6 +2,8 @@
     global:
         InitSocketTcti;
```

Comment:
Should the InitSocketTcti be private now?

Reply:
Still in use by the tools, tabrmd and others down stream. See previous comment on device TCTI.